Tests/ApolloCodegenTests/Frontend/DocumentParsingAndValidationTests.swift

```diff
@@ -124,7 +124,7 @@ class DocumentParsingAndValidationTests: XCTestCase {
     XCTAssertEqual(document.definitions.count, 3)
     
     let validationErrors = try codegenFrontend.validateDocument(schema: schema, document: document)
-    XCTAssertEqual(validationErrors, [])
+    XCTAssertEqual(validationErrors.count, 0)
```

Comment:
also doable as `XCTAssertTrue(validationErrors.isEmpty)` but whatever you feel is clearer.